src/components/Search/Search.js

```diff
@@ -7,16 +7,55 @@ const Platform = require("lib/platform");
 const Search = React.createClass({
   getInitialState: function () {
```

Comment:
this is a react-specific function that gets called internally so can't be a getter